.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:17.12.1-ce
+    working_directory: ~/ecs-lambda-runner
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          docker build -t developmentseed/cumulus:lambda-runner .
+          docker login -u scisco -p $DOCKER_PASSWORD
+          docker push developmentseed/cumulus:lambda-runner
```

Comment:
let's push this to the `cumuluss` org on docker hub?

Reply:
to be exact, push it to: `cumuluss/cumulus-ecs-task`

Reply:
Also, what is our plan for versioning here? I think we should not release with `latest` tag and start versioning this image.